scripts/populate.mjs

```diff
@@ -498,32 +422,17 @@ for await (let account of accounts.values()) {
     announcement: profileAnnouncement,
   } = await storeAnnouncement(profile, account.id, wallet);
 
-  let hashText = profileHash;
-  const announcements = [profileAnnouncement];
-
-  if (account.text) {
-    // create a note
-    const content = core.activityContent.createNote(
-      `${account.text} \n--from ${account.id}`,
-      { attachment: account.attachment }
-    );
-    content.published = new Date().toISOString();
-
-    const {
-      hash: contentHash,
-      announcement: noteAnnouncement,
-    } = await storeAnnouncement(content, account.id, wallet);
-
-    hashText += contentHash;
-    announcements.push(noteAnnouncement);
-  }
+  const {
+    hash: contentHash,
+    announcement: noteAnnouncement,
+  } = await storeAnnouncement(content, account.id, wallet);
 
-  const hash = web3.keccak256(hashText);
+  const hash = web3.keccak256(profileHash + contentHash);
 
-  const batchData = await core.batch.createFile(
-    hash + ".parquet",
-    announcements
-  );
+  const batchData = await core.batch.createFile(hash + ".parquet", [
+    profileAnnouncement,
+    noteAnnouncement,
+  ]);
```

Comment:
not sure what these changes are. should I roll them back?

Reply:
wonder if we want to make a make a copy of the populate script instead of changing the populate script.
Call the new one... populate_demo_2021_08.mjs ? or something like that?